joint_trajectory_controller/test/joint_trajectory_controller.test

```diff
@@ -35,5 +35,6 @@
   <test test-name="joint_trajectory_controller_test"
         pkg="joint_trajectory_controller"
         type="joint_trajectory_controller_test"
-        time-limit="85.0"/>
+        time-limit="85.0"
+        args="--gtest_filter=-JointTrajectoryControllerTest.jointVelocityFeedForward"/>
```

Comment:
Why do you exclude this test?

Reply:
I guess this test file is for the position interface only?
Since the code gets built twice anyway, It could be an option to add macro switch to emphasize that this test won't get run for the 

At least a comment should be added.

Reply:
> I guess this test file is for the position interface only?

Exactly.

> Since the code gets built twice anyway

Yeah, not very happy about this either. I'll see if I can come up with some way to only build them once.

And I'll add some comment anyways.

Reply:
>I'll see if I can come up with some way to only build them once.

Something like
`target_compile_definitions(joint_trajectory_controller_vel_test PRIVATE TEST_VELOCITY_FF=1)`?

Reply:
IMHO building it twice is not too bad.

Building it just once would require (?) to:
1. move tests (without FF) into a library
2. expose text fixture class as header (in test directory)
2. add  source file for the main function
3. `add_rostest_gtest` with main, linked to lib
3. `add_rostest_gtest` with main + ff-test, linked to lib

Reply:
Finally went for the `#ifdef TEST_VELOCITY_FF` check. Is it more or less what you had in mind, @ipa-mdl ?

Reply:
> Finally went for the #ifdef TEST_VELOCITY_FF check. Is it more or less what you had in mind,

It's exactly what I had in mind :D